internal/parse/arguments.go

```diff
@@ -0,0 +1,28 @@
+package parse
+
+import (
+	"flag"
+)
+
+type Options struct {
+	ReverseFlag bool
+	Files       []string
+}
+
+func optionFlags(ops *Options) {
+	flag.BoolVar(&(ops.ReverseFlag), "reverse", false, "reverse the result of comparisons")
+	flag.BoolVar(&(ops.ReverseFlag), "r", false, "reverse the result of comparisons")
+	flag.Parse()
+}
+
+func ArgumentParsing() *Options {
```

Comment:
`Arguments` would be a better name IMHO. In such a case you'll end up calling `parse.Arguments()` that's quite self explanatory.

Reply:
Also true, it was changed already